content/docs/user-guide/programming/components/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
-description: Learn how to create and reflect components in C++ using the O3DE Component Entity System.
-linktitle: Component Development
+linkTitle: Component Development
 title: Programmer's Guide to Component Development
+description: Learn how to create and reflect components in C++ using the O3DE Component Entity System.
```

Comment:
```suggestion
description: Learn how to create and reflect components in C++ and Python using the O3DE Component Entity System.
```
Python too?

Reply:
Reflecting methods to Python are not currently covered in this section. I would leave this out.

Reply:
I know O3DE supports tools and plugins written in Python. I'm not entirely sure about O3DE components, I'll ask sig-build in Discord.

Reply:
Just saw @willihay's comment 🙂 - Agreed, leave Python out. 
To add to the explanation, O3DE doesn't run Python at runtime, only in-editor.